adam-cli/src/main/scala/org/bdgenomics/adam/cli/ADAM2Vcf.scala

```diff
@@ -52,9 +52,14 @@ class ADAM2VcfArgs extends Args4jBase with ParquetArgs {
   @Args4jOption(required = false, name = "-coalesce", usage = "Set the number of partitions written to the ADAM output directory")
   var coalesce: Int = -1
 
-  @Args4jOption(required = false, name = "-sort_on_save", usage = "Sort the VCF output.")
+  @Args4jOption(required = false, name = "-sort_on_save", usage = "Sort the VCF output by contig index.")
   var sort: Boolean = false
 
+  @Args4jOption(required = false,
+    name = "-sort_lexicographically_on_save",
+    usage = "Sort the VCF output by lexicographic order. Conflicts with -sort_on_save.")
```

Comment:
Is there any possibility of a third sort method?  If so, maybe separate `-sort_on_save` and `-sort_by [method]` arguments would be preferrable